web/installer

```diff
@@ -1121,6 +1122,8 @@ class Installer
             }
             // Clean up stuff we created
             $this->uninstall();
+        } elseif ($this->tmpCafile) {
```

Comment:
why `elseif` ? Shouldn't we always clean the CA file when we have it ?

Reply:
We don't always have it though. The packaged CA data is only used if `--cafile` has not been used and getSystemCaRootBundlePath() can't find the system bundle.

Reply:
I'm not asking to remove the condition entirely. I'm asking to make that a `if` rather than a `elseif` so that it always gets cleaned in case it was used, even if other cleanup was done before in the method

Reply:
But it always get cleaned up. Could you explain why this is necessary.

Reply:
If the install fails, you enter `if (!$result)` above, and the elseif is never reached, so no clean up. If it was an if it would clean up still. I think that's what @stof is saying :)

Reply:
Okay. If the install fails we enter `if (!$result)` and everything we have installed, including `$this->tmpCafile` if it has been created, gets removed. There is even a highly technical (!) comment explaining this: `// Clean up stuff we created`

If the install succeeds we need to delete just one of the items that has been installed ...